### PR TITLE
[ADD] web_switch_company_warning : display a message saying it's time…

### PR DESCRIPTION
**Pull request by hparfr**
… to reload the page

### CODE REVIEW

File: web_switch_company_warning/readme.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+{project_repo}/issues/new?body=module:%20
+{module_name}%0Aversion:%20
+{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```
Comment (petrus-v) [inline web_switch_company_warning/readme.rst]: Hi!
Can you fill placeholder {project_repo} => web, and so on...?

File: web_switch_company_warning/static/src/js/switch_company_warning.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+openerp.web_switch_company_warning = function (instance) {
+
```
Comment (petrus-v) [inline web_switch_company_warning/static/src/js/switch_company_warning.js:4]: Can you use `"use strict";` as [recommended ](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#javascript)?

**Comment by hparfr** [comment]
@petrus-v Thanks for your suggestions.

**Comment by legalsylvain** [comment]
tried on rubot (all version), i have a JS bug.
web_switch_company_warning

**Comment by petrus-v** [comment]
:+1:

Tested on runbot with
FF 41.0 and
Chromium 45.0.2454.85

**Comment by laetitia-gangloff** [comment]
:+1:  (functionnal only)

Tested on runbot with 
google chorme 43.0.2357.130 (64-bit)

**Comment by bealdav** [comment]
:+1: Tested on runbot
Chrome Version 45.0.2454.85 Ubuntu 15.04 (64-bit)
Firefox 41.0

@legalsylvain could you make another test to guess what is the problem (empty cache, etc)

impossible to reproduce your bug

**Comment by legalsylvain** [comment]
I guess this commit https://github.com/hparfr/web/commit/861ad87e7e15452c98e9d35604b56cb54c4d0a1c fixed the bug mentionned in https://github.com/OCA/web/pull/233#issuecomment-144326205.

No code review. Tested on runbot. Works as expected. Thanks !

:+1:
